nuxt.config.js

```diff
@@ -164,6 +164,7 @@ export default defineNuxtConfig({
     '~/plugins/vueClipboard',
     '~/plugins/vueSocialSharing',
     '~/plugins/vueTippy',
+    '~/plugins/vue-query-plugin',
```

Comment:
Nuxt 3 auto imported plugin, I would rename file to `vue-query.client.ts`

```suggestion
```

Reply:
i did try to remove the line and it didn't work.
i'll try again with your name suggestion

Reply:
yeah... not working with that line :shrug:

Reply:
Alright, I can't wait to migrate from nuxt-bridge...